experimental/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to experimental packages in this project will be documented
 
 ### :boom: Breaking Change
 
+* feat(api-logs): Add delegating no-op logger provider [#4861](https://github.com/open-telemetry/opentelemetry-js/pull/4861) @hectorhdzg
```

Comment:
[marylia]
> Hey Hector, I see that you marked this as "Breaking change", would you mind providing more info on that?

[Hector]
> this is a breaking change because the behavior of logs.getLoggerProvider will return a ProxyLoggerProvider instead of the actual registered logger provider, same as current behavior for tracerProvider

Is that really a breaking change? `logs.getLoggerProvider` returns an object of type `interface LoggerProvider` both before and after, so I wonder (and hope) this could be considered **not** a breaking change.

A breaking change isn't so big a deal for this package, because it is still 0.x. However, a breaking change for the delegating MeterProvider in https://github.com/open-telemetry/opentelemetry-js/pull/4858, which is part of the "api" package, would be a very big deal: there is a VERY strong disincentive to have a major version bump for the "api" package.

Reply:
@hectorhdzg We discussed in the SIG today (I brought it up). We agreed that we do not need to consider this a breaking change, because the exported types for these APIs are interfaces.

Reply:
Thanks @trentm would update changelog then